chain/x/supply/keeper.go

```diff
@@ -47,19 +49,24 @@ func (k WrappedSupplyKeeper) BurnCoins(
 	// Create the account if it doesn't yet exist.
 	acc := k.GetModuleAccount(ctx, moduleName)
 	if acc == nil {
-		return sdk.ErrUnknownAddress(fmt.Sprintf("module account %s does not exist", moduleName))
+		panic(sdkerrors.Wrapf(
```

Comment:
Panic?

Reply:
Yes. I'm making the behavior as similar to the original `BurnCoins` as possible.
https://github.com/cosmos/cosmos-sdk/blob/ibc-alpha/x/supply/keeper/bank.go#L130